routes/api/search.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+const express = require("express");
+
+const Controllers = {
+    Search: require("../../controllers/search.controller")
+};
+
+const Middleware = {
+    Validator: {
+        Search: require("../../middlewares/validators/search.validator")
+    },
+    parseBody: require("../../middlewares/parse-body.middleware"),
+    Search: require('../../middlewares/search.middleware')
+};
+
+module.exports = {
+    /**
+     * @api {get} /search/
+     * @apiName search
+     * @apiGroup Search
+     * @apiVersion 0.0.8
+     *
+     * @apiParam (query) {String} model the model to be searched
+     * @apiParam (query) {Array} q the query to be executed
+     *
+     * @apiSuccess {String} message Success message
+     * @apiSuccess {Object} data Results
+     * @apiSuccessExample {object} Success-Response:
```

Comment:
You might also want to provide another example for when the message returns "no items found" or whatever